api/build.sh

```diff
@@ -19,6 +19,9 @@ set -ex
 export ENV=local
 pwd=`pwd`
 
+VERSION=$(cat ./api/VERSION)
```

Comment:
We also need go version, it is useful to debug problem, and if we add the git version, it would be better to add branch name. and build date is also useful.
You can refer [here](https://github.com/m3db/m3/blob/master/scripts/go-build-ldflags.sh).

Reply:
I agree with add go version. But why are branch name and build date useful, since we had git commit info.

In fact, I refer the [mosn](https://github.com/mosn/mosn/blob/2adbcbc57994288fa4bdeb86bf43973443233a0b/Makefile#L94)

Reply:
If here are many branch, how do you know to which branch the git version belong?

Reply:
Hi, I write the `abbreviated commit hash`.
If the commit hash is the same, we can consider the code is the same.

Reply:
Code is same is not enough, user could build it with different build tag and option(such as enable static link and disable cgo etc.).

Reply:
Got it. BTW, why we need to add the branch name?

Reply:
> If here are many branch, how do you know to which branch the git version belong?

For this, help us to locate at which branch git version is